.rubocop.yml

```diff
@@ -33,3 +33,24 @@ RescueModifier:
 
 HandleExceptions:
   Enabled: false
+
+Metrics/BlockLength:
+  Enabled: true
+  ExcludedMethods:
+    - class_methods
+    - included
+  Exclude:
+    - test/**/*
+    - bin/**/*
```

Comment:
`bin` might not strictly be necessary for all of these.

Reply:
Do we want to take it out?